src/buffer/out/Row.cpp

```diff
@@ -435,10 +435,9 @@ OutputCellIterator ROW::WriteCells(OutputCellIterator it, const til::CoordType c
     return it;
 }
 
-bool ROW::SetAttrToEnd(const til::CoordType columnBegin, const TextAttribute attr)

```

Comment:
Suppressing whitespace changes is required: https://github.com/microsoft/terminal/pull/15566/files?w=1

Practically nothing of substance changed in this PR. 🙂